pkgs/top-level/aliases.nix

```diff
@@ -1253,6 +1253,7 @@ mapAliases ({
   terraform_1_0 = throw "terraform_1_0 has been renamed to terraform_1"; # Added 2021-12-08
   terraform_1_0_0 = throw "terraform_1_0_0 has been renamed to terraform_1"; # Added 2021-06-15
   tesseract_4 = throw "'tesseract_4' has been renamed to/replaced by 'tesseract4'"; # Converted to throw 2022-02-22
+  testVersion = throw "'testVersion' has been moved to 'testers.testVersion'"; # Added 2022-04-20
```

Comment:
Let's make this a proper alias, so that out-of-tree expressions can use the old location until all nixpkgs versions without the new location are end of life.

I've set up a little bit of code and process to support this. `lib.isInOldestRelease 2205` will evaluate to `true` when the time to start the migration comes. https://github.com/NixOS/release-wiki/pull/32 It's a new thing, so might need some clarification and some helper functions and whatnot.

Reply:
changed it to a alias